cmd/gossamer/configcmd.go

```diff
@@ -80,13 +80,14 @@ func makeNode(ctx *cli.Context) (*dot.Dot, *cfg.Config, error) {
 	log.Info("🕸\t Configuring node...", "datadir", fig.Global.DataDir, "protocolID", string(gendata.ProtocolId), "bootnodes", fig.P2p.BootstrapNodes)
 
 	// TODO: BABE
+	sendChan := make(chan []byte)
```

Comment:
'send' doesn't make sense in this context, please rename to `coreToP2p` and `msgChan` to `p2pToCore`.